### PR TITLE
[Hexagon] Enable running CI tests via simulator

### PR DESCRIPTION
**Pull request by kparzysz-quic**
Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

**Comment by kparzysz-quic** [comment]
cc: @mehrdadh @masahi

File: tests/scripts/task_python_hexagon.sh [hunk 1]
```diff
@@ -23,6 +23,15 @@ source tests/scripts/setup-pytest-env.sh
 
 make cython3
 
-# unset because hardware does not exist in CI.
-unset ANDROID_SERIAL_NUMBER
-run_pytest ctypes python-contrib-hexagon-no-hwardware tests/python/contrib/test_hexagon/test_launcher.py
+export TVM_TRACKER_PORT=9190
```
[the remark this answers is not in the document]
Comment (kparzysz-quic) [reply tests/scripts/task_python_hexagon.sh:26]: I'm not sure if I understand---do you mean creating another shell script (essentially a copy of this one, but with changes for simulator) and leaving this one unchanged?

[the remark this answers is not in the document]
Comment (mehrdadh) [reply tests/scripts/task_python_hexagon.sh:26]: yes. The idea is to keep device specific setup separate and keep this script only for running the hexagon python tests.
@areusch might have better suggestions here.

[the remark this answers is not in the document]
Comment (kparzysz-quic) [reply tests/scripts/task_python_hexagon.sh]: Got it.

**Review by masahi — APPROVED** [review]

**Comment by masahi** [comment]
@kparzysz-quic please resolve the conflict.

File: tests/python/contrib/test_hexagon/test_launcher.py [hunk 2]
```diff
@@ -56,23 +63,24 @@ def test_add(android_serial_number, tvm_tracker_host, tvm_tracker_port):
...
         "rpc_tracker_port": tvm_tracker_port,
-        "rpc_server_port": 7070,
+        "rpc_server_port": 7070,  # See note at the beginning of the file
     }
     launcher = HexagonLauncher(serial_number=android_serial_number, rpc_info=rpc_info)
     launcher.upload(dso_binary_path, dso_binary)
     launcher.start_server()
 
     with launcher.start_session() as sess:
+        assert False
```
Comment (kparzysz-quic) [inline tests/python/contrib/test_hexagon/test_launcher.py:73]: *facepalm*

**Pull request merged**